app/notifications/followup_resolved_notification.rb

```diff
@@ -22,10 +22,20 @@ def title
   end
 
   def message
-    t(".message", created_by_name: params[:created_by_name])
+    t(".message", created_by_name: created_by_name)
   end
 
   def url
     edit_case_contact_path(params[:followup][:case_contact_id], notification_id: record.id)
   end
+
+  private
+
+  def created_by_name
```

Comment:
duplicate code block- can we combine them?

Reply:
Sure